lib/functions.php

```diff
@@ -14,8 +14,7 @@
      *
      * @template TReturn
      *
-     * @param callable(mixed ...$args):(\Generator<mixed,Promise|ReactPromise|array<array-key, Promise|ReactPromise>,mixed,Promise<TReturn>|ReactPromise|TReturn>|Promise<TReturn>|ReactPromise|TReturn)
-     *     $callback
+     * @param callable(mixed ...$args):(\Generator<mixed,Promise|ReactPromise|array<array-key, Promise|ReactPromise>,mixed,Promise<TReturn>|ReactPromise|TReturn>|Promise<TReturn>|ReactPromise|TReturn) $callback
```

Comment:
The phpstan docblock parser didn't like `$callback` being ona new line for some reason.

Reply:
Could you please open an phpstan issue for that? Phpstorm automatically formats it like that for me.

Reply:
Hmm, it works now it seems.

Reply:
PhpStorm doesn't understand this syntax at all so I'm not surprised it breaks on that.

Reply:
@ondrejmirtes: PhpStorm doesn't break, it just reformats too long lines like that.

Reply:
@kelunik Yeah but it won't be able to understand the method accepts `callable`. Maybe in this case yes, because the type starts with `callable`, but definitely not in some more complex cases.